static/ansi-to-html.ts

```diff
@@ -43,22 +43,22 @@ const defaults: AnsiToHtmlOptions = {
 
 function getDefaultColors(): ColorCodes {
     const colors: ColorCodes = {
-        0: '#000',
-        1: '#A00',
-        2: '#0A0',
-        3: '#A50',
-        4: '#00A',
-        5: '#A0A',
-        6: '#0AA',
-        7: '#AAA',
-        8: '#555',
-        9: '#F55',
-        10: '#5F5',
-        11: '#FF5',
-        12: '#55F',
-        13: '#F5F',
-        14: '#5FF',
-        15: '#FFF',
+        0: 'var(--terminal-black)',
```

Comment:
Now I understand the question in Discord :)

Reply:
Seemed to be the way to go over having 32 class names :)